src/main/java/chess/dao/PiecesDao.java

```diff
@@ -0,0 +1,127 @@
+package chess.dao;
+
+import chess.domain.Board;
+import chess.domain.Movement;
+import chess.domain.Position;
+import chess.domain.piece.abstractPiece.Piece;
+import chess.domain.piece.character.Kind;
+import chess.domain.piece.character.Team;
+import chess.exception.DataAccessException;
+import chess.exception.InvalidGameRoomException;
+import chess.util.PositionConverter;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public final class PiecesDao {
+    private static final String TABLE_NAME = "pieces";
+
+    public void addAll(Board board, String roomName, Connection connection) {
+        try {
+            for (Entry<Position, Piece> piece : board.getPieces().entrySet()) {
+                PreparedStatement statement = connection.prepareStatement(
+                        "INSERT INTO " + TABLE_NAME
+                                + "(room_name, position, team, kind, is_moved)"
+                                + " VALUES (?, ?, ?, ?, ?)");
```

Comment:
```suggestion
                        PreparedStatement statement = connection.prepareStatement("""
                    INSERT INTO pieces (room_name, position, team, kind, is_moved) 
                    VALUES (?, ?, ?, ?, ?)
                    """
                );
```
쌍따옴표 세개를 사용해서 가독성을 더 올릴 수 있을것 같네요
(정확한 명칭이 기억은 안나네요)

Reply:
좋은 정보 감사합니다~